bindings/protocols/coap/index.html

```diff
@@ -142,23 +142,35 @@ <h3>Content Negotiation</h3>
                 The CoAP Accept option is used by clients to request a particular content format, while the <code>Content-Format</code> option is used by clients and servers to indicate the content format of the representation in requests and responses, respectively.
             </p>
             <p>
-                The ... can be used to provide a list of one or more content formats that the Thing is able to provide in a CoAP response for the operation.
-                The Consumer selects the most suitable one and sets the <code>Accept</code> option in the request accordingly.
+                Since mapping the string-based <code>contentType</code> and <code>contentCoding</code> fields to a numeric Content-Format can be prone to error, this document defines a vocabulary for directly defining the Content-Format(s) associated with an affordance.
+                To do so, the term <code>cov:contentFormat</code> can be used in the same places as <code>contentType</code> and <code>contentCoding</code>.
```

Comment:
`contentType` must be present otherwise the consumer must assume json and this will cause problems